common/source/docs/common-network.rst

```diff
@@ -0,0 +1,118 @@
+.. _common-network:
+
+===========================
+ArduPilot Network Interface
+===========================
+
+As of firmware 4.5 and later, ArduPilot provides a network interface framework to allow local and wide-area network connections. Several later generation autopilots, like Pixhawk6X and CubePilot CubeRed, provide Ethernet physical layer interfaces, which allows ArduPilot to connect to vehicle peripherals, data servers, and even the wide area network via IP using transport layer UDP or TCP protocols.
```

Comment:
Give PPP, perhaps add. "PPP networking can be used even on boards that don't have physical Ethernet interfaces, and supports all of the features of the network interface framework."

Reply:
see below